rollup.config.js

```diff
@@ -28,26 +44,26 @@ export default [
         format: 'cjs',
         exports: 'named',
       },
+    ],
+    plugins: [...plugins],
+  },
+  {
+    input: 'src/instantsearch.js',
+    external: [
+      'algoliasearch-helper',
+      'instantsearch.js/es',
+      'instantsearch.js/es/connectors',
```

Comment:
Do we need this line?

Reply:
we import directly from es/connectors, so yes as far as I can tell. It's the same as before in any case

Reply:
Yes, I did not touch this. It is the same configuration as before.

Reply:
Yeah I saw it was the same before, but got curious if `/connectors` has any effect while we already have `instantsearch.js/es`. And we import things like `instantsearch.js/es/lib/xxx` but they're not here, either.

I removed `/connectors` and built. And the size of cjs and esm seem to be the same.
But I'm not sure about anything since I haven't dug in depth. Not a blocker at all here.